web_widget_darkroom/static/lib/darkroomjs/core/darkroom.js

```diff
@@ -0,0 +1,356 @@
+/**
+*    Copyright 2013 Matthieu Moquet
+*    Copyright 2016-2017 LasLabs Inc.
+*    License MIT (https://opensource.org/licenses/MIT)
+**/
```

Comment:
Hmmm I wonder what version darkroom this is? We should update then add the version number into the header